include/cppjni/method_call/method_caller.hpp

```diff
@@ -4,7 +4,7 @@
 
 #include <boost/hana/tuple.hpp>
 #include <boost/hana/zip_with.hpp>
-#include <boost/static_assert.hpp>
+//#include <boost/static_assert.hpp>
```

Comment:
Ouch - my bad that this include stayed in code : /. Still, it's a dead code, please remove that.